BankingApplication/src/main/java/com/ark/bankingapplication/views/Dashboard.java

```diff
@@ -117,33 +153,37 @@ public void setBank(String bank) {
         this.bankId = bank;
     }
 
+    /**
+     * Set the sessionKey
+     *
+     * @param sessionKey string
+     */
     public void setSessionKey(String sessionKey) {
         this.sessionKey = sessionKey;
-        System.out.println(this.sessionKey);
     }
 
     public void setCustomer(Customer customer) {
         this.customer = customer;
     }
 
     public void setLogo() {
+        URL iconUrl = this.getClass().getResource("images/" + this.bankId + "-ICON.png");
         String path = new File("BankingApplication/src/main/java/com/ark/bankingapplication/views/images/" + this.bankId + "-ICON.png").getAbsolutePath();
         File file = new File(path);
-        Image image = new Image(file.toURI().toString());
+        Image image = new Image(iconUrl.toString());
         this.bankLogo.setImage(image);
         this.setBankNameLabel(this.bankId);
 
 
     }
 
     private void doLogout() {
-        boolean logout = this.controller.logout(this.sessionKey);
-        if (!logout) {
-            showWarning("Fout bij uitloggen", "Uitloggen mislukt, probeer het later nog eens!");
-        } else {
+        try {
+            boolean logout = this.controller.logout(this.sessionKey);
```

Comment:
Er wordt hier niets met doe boolean gedaan. Bewust?

Reply:
Nee was meer om te printen om te kijken wat er gebeurde... maakt verder ook geen fuck uit als hij de sessie niet meer weet te vinden, je gaat uitloggen dus als de sessie kwijt is is dat alleen maar prima!!